src/tests/tests.ts

```diff
@@ -1492,7 +1494,7 @@
 			wait(100);
 			var t2 = time();
 			var d = t2 - t1;
-			if d < 90 or d > 130 then print("wrong");
+			if d < 50 or d > 200 then print("wrong");
```

Comment:
Is there a specific reason that you changed this? Is the timing less reliable when running in node?

Reply:
Yes and now. Tests are run async. This speeds up execution without needing to change `Interpreter.chunk`.  That means many more tests are run at once. This also means that depending on scheduling the timing might be off.